--- origsrc/grace-5.1.23/auxiliary/Makefile	2013-01-01 19:13:34.279625000 -0600
+++ src/grace-5.1.23/auxiliary/Makefile	2013-01-01 19:18:28.821431400 -0600
@@ -22,7 +22,7 @@ distclean : clean
 devclean : distclean
 
 convcal$(EXE) : convcal.c
-	$(CC) $(CFLAGS0) $(CPPFLAGS) -Wl,-z,relro -Wl,-z,now -o $@ convcal.c $(NOGUI_LIBS)
+	$(CC) $(CFLAGS0) $(CPPFLAGS) -o $@ convcal.c $(NOGUI_LIBS)
 
 install : $(AUXILIARIES) $(PROGRAMS) $(SCRIPTS)
 	$(MKINSTALLDIRS) $(DESTDIR)$(GRACE_HOME)/auxiliary
